packages/ui/src/components/vuestic-components/va-app-bar/VaAppBar.demo.vue

```diff
@@ -14,7 +14,7 @@
       <div class="frame">
         <div class="display">
           <va-app-bar color="secondary">
-            <AppBarTestContent />
+            <AppBarTestContent color="" />
```

Comment:
Did you forget to put color here or remove the attribute?

Reply:
No, this is done on purpose to override the default value. After that, the buttons inside the component will not be customized.